xbmc/pvr/timers/PVRTimerInfoTag.cpp

```diff
@@ -612,6 +640,12 @@ bool CPVRTimerInfoTag::UpdateEntry(const CPVRTimerInfoTagPtr &tag)
   if (m_strSummary.empty())
     UpdateSummary();
 
+  m_iActiveChildTimers = 0;
+  m_bHasChildConflictNOK = false;
+  m_bHasChildRecording = false;
+  m_bHasChildErrors = false;
+  // Updated in CPVRTimers::UpdateEntries which should run after this...
```

Comment:
This comment is a clear sign of bad design. Separation of concerns! Do it like I suggested (the new CPVRTimerInfoTag method) and comments like this can go away. ;-)